src/layouts/IndexPage.jsx

```diff
@@ -49,9 +43,49 @@ export const IndexPage = () => {
   const classes = useStyles();
   const [, dispatch] = useContext(GlobalStateContext);
 
+  const [newsList, setNewsList] = useState([]);
+
   useEffect(() => {
-    announce('Pagina caricata ' + name);
+    (async () => {
+      try {
+        await fetch(`https://raw.githubusercontent.com/tuxolino/testapi/main/data/NC002.json`)
+          .then((res) => res.json())
+          .then((data) => {
+            setNewsList(data.windows);
+          });
+      } catch (error) {
+        console.log(error);
+      }
+    })();
   }, []);
+
+  // API rest to get SF TOKEN
```

Comment:
what is it for?

Reply:
It's an error, I removed the comment block